docs/fsharp/get-started/installing-fsharp.md

```diff
@@ -0,0 +1,64 @@
+--
```

Comment:
The metadata is appearing as text on the built page. There should be three hyphens instead of two to delimit the metadata section.